server/dao/listing.js

```diff
@@ -42,7 +42,7 @@ class ListingDao {
     return rows && rows.length >= 1 ? rows[0] : null;
   };
 
-  updateListing = async (id, body) => {
+  updateListing = async (id, lid, body) => {
```

Comment:
Personally I would prefer id to be listingid and uid to be userid, since this is in ListingDao.